src/lib.rs

```diff
@@ -800,23 +801,27 @@ impl<T: Counter> Histogram<T> {
     ///
     /// println!("{:?}", hist.iter_percentiles(1).collect::<Vec<_>>());
     ///
-    /// assert_eq!(perc.next(), Some((hist.value_at_percentile(0.01), 0.01, 1, 1)));
+    /// assert_eq!(perc.next(), Some(IterationValue { value: hist.value_at_percentile(0.01),
+    ///     percentile: 0.01, count_at_value: 1, count_since_last_iteration: 1 }));
```

Comment:
I'm happy to reformat; I only went with compact-but-ugly to save relatively boring lines of source. Would you prefer more like this:

```
    /// assert_eq!(perc.next(), Some(IterationValue { 
    ///     value: hist.value_at_percentile(0.01),
    ///     percentile: 0.01, 
    ///     count_at_value: 1,
    ///     count_since_last_iteration: 1
    /// }));
￼ ```

Given the documentation nature of these asserts, I think it's nice for the user to see explicitly what data is available at each point in the iteration, but if we move to non-`pub` members then we'll have `new` here anyway.